packages/hydrogen/src/cart/queries/cartAttributesUpdateDefault.doc.ts

```diff
@@ -0,0 +1,33 @@
+import {ReferenceEntityTemplateSchema} from '@shopify/generate-docs';
+
+const data: ReferenceEntityTemplateSchema = {
+  name: 'cartAttributesUpdateDefault',
+  category: 'utilities',
+  isVisualComponent: false,
+  related: [],
+  description:
+    'Creates a function that accepts an array of [AttributeInput](/docs/api/storefront/2023-04/input-objects/AttributeInput) and updates attributes to a cart',
```

Comment:
Maybe for each of these we could add a note that they are to be used within an action. Maybe even if the examples themselves just show them in an action?

```suggestion
    'Creates a function that accepts an array of [AttributeInput](/docs/api/storefront/2023-04/input-objects/AttributeInput) and updates attributes to a cart',
```